### PR TITLE
Update puppet-foreman_scap_client to 0.3.20

### PR DESCRIPTION
**Pull request by ares**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.21
* [ ] 1.20

See Foreman's [plugin maintainer documentation](https://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

**Comment by ekohl** [comment]
Ping

**Comment by ares** [comment]
Sorry about the delay, I excluded spec files (we don't ship tests elsewhere I believe), I hope that's what you meant by `%spec`. I kept it as separate commit, feel free to squash or ask me to do it, if it turns green.

**Comment by ekohl** [comment]
Looks like it failed but the tests have been rotated away already [test rpm]

File: packages/plugins/puppet-foreman_scap_client/puppet-foreman_scap_client.spec [hunk 1]
```diff
@@ -42,8 +42,12 @@ cp -rp . %{buildroot}/%{puppet_foreman_scap_client_dir}/
 %{puppet_foreman_scap_client_dir}/lib
 %{puppet_foreman_scap_client_dir}/manifests
 %{puppet_foreman_scap_client_dir}/templates
+%exclude {puppet_foreman_scap_client_dir}/spec
```
Comment (ekohl) [inline packages/plugins/puppet-foreman_scap_client/puppet-foreman_scap_client.spec]: BUILDSTDERR:     File must begin with "/": {puppet_foreman_scap_client_dir}/spec

```suggestion
%exclude %{puppet_foreman_scap_client_dir}/spec
```

**Pull request merged**